### ISSUE TITLE
Taxonomies & 'ugly' URLs don't get along

### ISSUE THREAD

**Issue by eadmund** [alanorth/hugo-theme-bootstrap4-blog#85]
With `uglyurls = true`, links to taxonomy pages are to e.g. `tags/` rather than `tags.html`.

**Comment by alanorth** [comment]
You're right, it seems to be an issue that several people have reported already, for example https://github.com/gohugoio/hugo/issues/1989. I will look at some of the proposed solutions.
